### ISSUE TITLE
Process updates and release announcements

### ISSUE THREAD

**Issue by yotam180** [yotam180/WhatsappAPI#33]
Hello, curious folks!

Some have consulted me about how to use the API. 
At the moment, the API is yet to be functional, but I promise when I settle down and finish a first version I'll announce and post it in this thread. 

If you want to get notified, just subscribe to this thread :smile: 

Cheers.

**Comment by yotam180** [comment]
Alright, something is working... This project is becoming one huge spaghetti code.
I gotta maintain it organized.

**Comment by yotam180** [comment]
First beta version is ready [here](https://chrome.google.com/webstore/detail/whatsapp-api/klddgngdkdljhlhmjdllbdechlnmacik)! 

Documentation, however, is still work-in-progress. Sorry :man_shrugging:

**Comment by surfmore** [comment]
The link above is not working. Can you help please.

**Comment by yotam180** [comment]
@surfmore Yes. It was taken down for "copyrights" which the extension violated, but it was (hopefully) fixed.
You can build the chrome extension from sources. If you need help with that I can assist. Just open another issue rather than commenting here please :slightly_smiling_face:
